src/main/java/com/github/sergdelft/j2graph/DataGenerator.java

```diff
@@ -0,0 +1,104 @@
+package com.github.sergdelft.j2graph;
```

Comment:
can we put this in a `iclr20great` package?

Reply:
Will do

Reply:
Done